tests/ballerina-unit-test/src/test/resources/test-src/balo/test_projects/test_project_negative/.ballerina/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Unintentional commit?

Reply:
Added .gitignore to add the .ballerina directory